<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_setting_PC_controller</class>
 <widget class="QDialog" name="view_setting_PC_controller">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>297</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PC control desired theta --&gt; Configuration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>12</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select your controller type and fill the arguments</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="type_1_groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>42</y>
     <width>259</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>     Cascade Controller</string>
   </property>
   <widget class="QCheckBox" name="cascade_controller_checkBox">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="k1_label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>42</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>K1 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="k1_textBox">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>39</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="k2_textBox">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>39</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="k2_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>42</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>K2 : </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="type_2_groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>148</y>
     <width>259</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>     Leadlag Controller</string>
   </property>
   <widget class="QCheckBox" name="leadlag_controller_checkBox">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="a1_textBox">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>25</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="a1_label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>28</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>A1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="a2_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>28</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>A2 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="a2_textBox">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>25</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="b1_textBox">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>65</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="b1_label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>68</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>B1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="b2_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>68</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>B2 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b2_textBox">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>65</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="b3_textBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>65</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="b3_label">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>68</y>
      <width>21</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>B3 : </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
